tests/xharness/Jenkins/TestSelector.cs

```diff
@@ -180,6 +184,7 @@ void SelectTestsByModifiedFiles (int pullRequest)
 			SetEnabled (files, cecilPrefixes, "cecil", ref jenkins.IncludeCecil);
 			SetEnabled (files, dotnetFilenames, "dotnet", ref jenkins.IncludeDotNet);
 			SetEnabled (files, msbuildFilenames, "msbuild", ref jenkins.IncludeMSBuild);
+			SetEnabled (files, xharnessPrefix, "all", ref jenkins.IncludeAll);
```

Comment:
Won't this (try to) run the device tests too?

Reply:
AFAIk it won't, but we shall see in this run :)

Reply:
shows it does not <img width="424" alt="Screenshot 2021-05-20 at 17 32 15" src="https://user-images.githubusercontent.com/2190086/119051559-5442ea00-b991-11eb-8f84-1dfdb486065d.png">

The main reason is that the all is overriden by the skip-device-tests from the make. And auto selected tests are override by labels.